src/Default/default.ts

```diff
@@ -208,8 +208,8 @@ export class DefaultContainer extends WebContainerBase {
         const uuid = window[DefaultContainer.windowUuidPropertyKey] = Guid.newGuid();
         windows[uuid] = window;
 
-        // Attach unload handler to window to cleanup reference on global windows object
-        window.addEventListener("unload", (event) => delete windows[uuid]);
+        // Attach beforeunload handler to window to cleanup reference on global windows object
+        window.addEventListener("beforeunload", (event) => delete windows[uuid]);
```

Comment:
I'm not sure about now but it used to be the case that to cover some browsers, it was worth subscribing to both events.  Not sure if that's still the case, though.

Reply:
@dovie this works in IE 11 too.

Reply:
@psmulovics You got me there.

Reply:
Give me an event (and no jquery)

Reply:
```
<script type="text/javascript" for="window" event="onunload">
  alert("The onunload event fired for the window object.");
</script>
```
This is supposed to work in IE11 (this is what jquery does under the hood for you)

Reply:
That's exactly what the code was.  Chrome fires unload after load.

Reply:
I can attach to unload within beforeunload